packages/server/safeMutations/safeArchiveTeam.ts

```diff
@@ -14,18 +13,9 @@ const safeArchiveTeam = async (teamId: string, dataLoader: DataLoaderWorker) =>
     .filter({isNotRemoved: true})('userId')
     .run()
   await removeUserTms(teamId, userIds)
-  const updates = {
-    isArchived: true,
-    updatedAt: new Date()
-  }
   const users = await Promise.all(userIds.map((userId) => dataLoader.get('users').load(userId)))
   const [rethinkResult, pgResult] = await Promise.all([
     r({
-      team: r
-        .table('Team')
-        .get(teamId)
-        .update(updates, {returnChanges: true})('changes')(0)('new_val')
```

Comment:
The system does not set `updatedAt` in Postgres compared to RethinkDB.

Reply:
I think instead of adding the `updatedAt` times in code here where they could be forgotten, we should add a trigger like we did for "User" table: https://github.com/ParabolInc/parabol/blob/d05cd9b6c7ae3e2395e3c3e9de25d4786fa1184e/packages/server/postgres/migrations/1638368905980_userUpdatedAtTrigger.ts#L8

Reply:
As a workaround for the current case, it would be good if we could:
1) @shapel allow some delta when comparing dates
2) @tianrunhe pull the dates from rethink and update PG
Let me know whether or not that works for you.

Reply:
@Dschoordsch thanks for review.
Of course, we could add a trigger to the "Team" table, but I see that this is still not a common pattern across the system, which can be confusing if the DB and the web server might have time synchronization problems. 
It's always better to have one source of truth...
I could add such a trigger to all tables in a separate PR (to make sure we have the same approach throughout the system).

Regarding to possible workaround...
According to the examples provided by @tianrunhe, there are records in the system with dates that differ by about a year. And I guess they can't potentially be updated in the near function.  Therefore I suggest adding the `updateAt` field to the ignore list when comparing records for the time being.

Reply:
Adding "updatedAt" to the ignored fields is fine for me. We should still try to correct our data while we have the chance, but I leave that up to @tianrunhe to decide.

I know the "set_updatedAt" trigger is not a common practice yet, but I'd like to get there as it's easy to forget updating that field, as you've noticed. This would include removing all manual updates to that field as well.